.github/workflows/cargo_publish_dry_run.yml

```diff
@@ -15,26 +16,12 @@ jobs:
     steps:
       - uses: actions/checkout@v4
       - name: Install stable Rust
-        uses: actions-rs/toolchain@v1
-        with:
-          toolchain: stable
-          profile: minimal
+        uses: dtolnay/rust-toolchain
```

Comment:
Why this change?

Reply:
cc https://github.com/pubgrub-rs/pubgrub/issues/164

Reply:
This action is used everywhere else, this was the only exception, and action-rs is unmaintained.